exonum/src/helpers/mod.rs

```diff
@@ -104,7 +104,9 @@ fn has_colors() -> bool {
 }
 
 fn format_time(time: SystemTime) -> String {
-    DateTime::<Local>::from(time).to_rfc2822()
+    DateTime::<Local>::from(time)
+        .format("%a, %e %b %Y %H:%M:%S%.3f %z")
```

Comment:
Maybe move this format string to a constant with comment describing what exactly it's doing?
With a brief look it looks confusing.